magento/magento2
# Make it possible to disable cross-sell on cart page

### Description
To be able to select whether to have the cross-sell active on the cart page an admin configuration is created under Stores > Configuration > Sales > Checkout > Shopping Cart.

The default for this setting is set to Yes (Show cross-sell items). This is for backwards compatibility and not have webshop owners accidentally not showing these items.

Because this is now sorted as the 3rd configuration in this section/group the extra configurations for the grouped product image and the configurable product image are now also increased.

This feature will make it possible to have this cross-sell turned off during special sales, A/B testing or in a specific store view.

### Manual testing scenarios
1. Set up a cross-sell
2. Add that product that has the cross-sell product configured to your cart
3. Go to the shopping cart
4. See that cross-sell is shown
5. Change the new setting 'Show Crosssell Items in the Shopping Cart' to 'No' _(setting is under Stores > Configuration > Sales > Checkout > Shopping Cart)_
6. Refresh affected caches
7. Go back to the shopping cart
8. See that cross-sell is not shown

### Contribution checklist
